packages/dev/sharedUiComponents/src/lines/sliderLineComponent.tsx

```diff
@@ -169,7 +170,15 @@ export class SliderLineComponent extends React.Component<ISliderLineComponentPro
                 />
                 <div className="slider">
                     <input
-                        className="range"
+                        className={
+                            "range" +
+                            (this.props.allowOverflow &&
+                            this.props.minimum &&
+                            this.props.maximum &&

```

Comment:
Do we really need these checks, as `minimum` and `maximum` are mandatory? Also, this will fail if one of the value is 0.